_data/tools.yml

```
@@ -1796,6 +1796,18 @@
   description: NodeJs OpenAPI 3 based documentation generator which sits between server APIs and anyone calling the APIs
   v3: true
 
+- name: OpenAPI AutoSpec
+  category:
+    - auto-generators
+    - documentation
+  language: JavaScript
+  link: https://www.portway.ai/
+  github: https://github.com/Adawg4/openapi-autospec
+  description: Proxy server that generates API specs for any app or website on localhost
```

We don’t call them specs even when the tool does. 

https://github.com/openapi-contrib/glossary?tab=readme-ov-file#api-description
```suggestion
  description: Proxy server that generates API descriptions for any app or website on localhost.
```

---

Will keep that in mind, cheers